src/object_store/src/object/mod.rs

```diff
@@ -922,5 +1105,136 @@ pub async fn build_remote_object_store(
     }
 }
 
+#[inline(always)]
```

Comment:
BTW. Should we build the strategy table on startup instead of generating every time?